tests/foreman/longrun/test_oscap.py

```diff
@@ -297,6 +315,8 @@ def test_positive_push_updated_content(self):
             satellite should get updated reports
 
         :CaseLevel: System
+
+        :BZ: 1420439
```

Comment:
s/1479413/1722475

Reply:
will add the other BZ aswell , the one mentioned is valid aswell